flow-report/test/topbar-test.tsx

```diff
@@ -34,6 +28,7 @@ let wrapper: FunctionComponent;
 let options: LH.FlowReportOptions;
 
 beforeEach(() => {
+  saveHtml.saveFile = defaultSaveFile;
```

Comment:
State should be restored in `afterEach`, else if the test happens to be run last it won't undo it's mess for the next test suite.